mlflow/utils/download_cloud_file_chunk.py

```diff
@@ -41,20 +41,11 @@ def main():
             download_path=args.download_path,
             http_uri=args.http_uri,
         )
-    except (ConnectionError, ChunkedEncodingError):
```

Comment:
There are many other possible exceptions that could occur

Reply:
Zero reservations here on catching all and retrying. I think we were just trying to avoid unnecessary broad retries, but, as you mention, it's pretty much impossible to maintain a list of error codes that could potentially be raised. This change LGTM :)

Reply:
I'd remove try-catch if we retry on any exception.